### PR TITLE
8894 - Dropdown Wrapper Excess Space

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This pull request will remove excess space on dropdown-wrapper class.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/8894

**Steps necessary to review your pull request (required)**:
- Pull this git repository, build, and run the app
- Go to http://localhost:4000/components/dropdown/example-clearable?theme=uplift&variant=light&layout=embedded
- Open `Developer Console`
- Open one dropdown
- Select and find the `dropdown-wrapper`
- It should not have excess space

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Review by glenlieorillo — DISMISSED** [review]
image

File: src/components/dropdown/dropdown.js [hunk 1]
```diff
@@ -253,8 +253,7 @@ Dropdown.prototype = {
     this.isWrapped = this.wrapper.length > 0;
 
     if (!this.isWrapped) {
-      // debugger;
```
Comment (ericangeles) [inline src/components/dropdown/dropdown.js:256]: Could you cleanup the code, @tjamesallen15?

**Review by jbrcna — DISMISSED** [review]
image

File: src/components/dropdown/dropdown.js (hunk 1 above)
Comment (tjamesallen15) [reply src/components/dropdown/dropdown.js:256]: thats already removed

**Review by ericangeles — APPROVED** [review]

**Comment by tmcconechy** [comment]
@tjamesallen15 looks like the tests need you to update a snap shot.

**Review by tmcconechy — APPROVED** [review]
Tests failing. Reruning this

**Review by tmcconechy — DISMISSED** [review]
Looks good @tjamesallen15 just please update the snapshot in the tests..

```
npx playwright test -u -- dropdown
```

**Review by yohannahbautista — APPROVED** [review]

**Comment by tmcconechy** [comment]
Still needs work on the tests...

**Review by tmcconechy — APPROVED** [review]

**Comment by tjamesallen15** [comment]
> Looks good @tjamesallen15 just please update the snapshot in the tests..
> 
> ```
> npx playwright test -u -- dropdown
> ```

I ran the test and this happen. @tmcconechy 

Image 9-5-24 at 2 04 PM

**Review by ericangeles — APPROVED** [review]

**Comment by tmcconechy** [comment]
@tjamesallen15 the command looked correct. Once you run it commit the result. But i tried to run it for you. Seems like some error in the git log so rerunning it (never seen that one before). So no real actions for you on this for now

**Pull request merged**